_tests/integration/log_filter_test_secrets.yml

```diff
@@ -1,4 +1,5 @@
 envs:
+- BITRISE_ENABLE_SECRET_FILTERING: true
```

Comment:
`ENABLE` is the same as setting an env to true or leave it as false. Please remove.